portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -25,24 +25,43 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  ArrayList<String> comments = new ArrayList<String>();
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> comments = new ArrayList<String>();
-    comments.add("Hello");
-    comments.add("How are you?");
-    comments.add("Greetings");
-
+    System.out.println("I do GET");
     //Convert comments arraylist to JSON
     String json = convertToJson(comments);
 
     response.setContentType("application/json;");
     response.getWriter().println(json);
   }
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    System.out.println("I do POST");
+    String comment = getComment(request);
+    comments.add(comment);
+    response.setContentType("text/html");
+    response.getWriter().println("Comment submitted!");
+
+    // Redirect back to the HTML page.
+    response.sendRedirect("/index.html");
+  }
+
   private String convertToJson(ArrayList<String> comments) {
+    System.out.println("I convert to json");
```

Comment:
System.out.println

Reply:
Fixed in https://github.com/anamatasanch/SPS-portfolio/pull/12/commits/d3073c67f9f36eb9cc31a991921c1cd19f8475cc